From 8d43fecf27f98969f59aeb63e61ec4f088849949 Mon Sep 17 00:00:00 2001
From: "yi.liu" <yi.liu@amlogic.com>
Date: Thu, 27 Apr 2023 12:02:05 +0800
Subject: [PATCH] amlbian: debian build error [1/1]

PD#SWPL-95583

Problem:
debian build error.

Solution:
debian build error.

Verify:
local

Signed-off-by: yi.liu <yi.liu@amlogic.com>
Change-Id: I556748d9c4b66c561984648d98d8fa24ae04e01d
---
 config/Debian/distribution-special-function | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config/Debian/distribution-special-function b/config/Debian/distribution-special-function
index cfbbd35..34d5e40 100755
--- a/config/Debian/distribution-special-function
+++ b/config/Debian/distribution-special-function
@@ -104,6 +104,7 @@ install_distribution_specific()
 
 post_install_distribution_specific()
 {
+	:
 }
 
 
-- 
2.25.1

